clients/rospy/src/rospy/core.py

```diff
@@ -155,52 +155,48 @@ def _frame_to_caller_id(frame):
     return pickle.dumps(caller_id)
 
 
-def _base_logger(msg, *args, **kwargs):
-
-    name = kwargs.pop('logger_name', None)
-    throttle = kwargs.pop('logger_throttle', None)
-    level = kwargs.pop('logger_level', None)
-    once = kwargs.pop('logger_once', False)
-    throttle_identical = kwargs.pop('logger_throttle_identical', False)
+def _base_logger(msg, args, kwargs, throttle=None,
```

Comment:
I don't see why `args` are `kwargs` can't continue to be passed as `*args` and `**kwargs`. What is the benefit wrapping them into a single parameter?